apps/sms/views/shared/style/composer.css

```diff
@@ -34,6 +34,7 @@ form[role="search"] button[type="submit"] + p > textarea {
   -moz-border-end: 0.1rem solid #ccc;
 
   font-size: 1.7rem;
+  font-family: "Firefox Emoji", sans-serif;
```

Comment:
__isssue:__ I see that with Emoji font applied, even if I use normal English keyboard, numbers are always displayed as fancy 0️⃣ 1️⃣ 2️⃣ 3️⃣ 4️⃣ 5️⃣ :)